src/main/java/de/chojo/repbot/dao/access/Gdpr.java

```diff
@@ -35,11 +38,12 @@ public List<RemovalTask> getRemovalTasks() {
 
     public void cleanupRequests() {
         builder()
-                .queryWithoutParams("""
-                                    DELETE FROM gdpr_log
-                                    WHERE received IS NOT NULL
-                                        AND received < NOW() - INTERVAL '90 DAYS';
-                                    """)
+                .query("""
+                       DELETE FROM gdpr_log
+                       WHERE received IS NOT NULL
+                           AND received < NOW() - INTERVAL ?;
```

Comment:
Any reason why this is not like `X::interval`?
```suggestion
                           AND received < NOW() - ?::INTERVAL;
```

Reply:
Because it's not the same.`::` is a cast, `Interval X` creates an interval value from a string.

Reply:
Both works essentially the same, but the cast syntax is currently recognized as a statement placeholder in intellij, so I switched for the better readable syntax to test those queries better